lib/aws/rails/middleware/ebs_sqs_active_job_middleware.rb

```diff
@@ -85,7 +85,15 @@ def sent_from_docker_host?(request)
       end
 
       def app_runs_in_docker_container?
-        @app_runs_in_docker_container ||= `[ -f /proc/1/cgroup ] && cat /proc/1/cgroup` =~ /docker/
+        @app_runs_in_docker_container ||= in_docker_containr_with_cgroup1? || in_docker_containr_with_cgroup2?
+      end
+
+      def in_docker_containr_with_cgroup1?
```

Comment:
Not sure if you noticed, but this should be `container` (you forgot the `e`)

Reply:
oh, it's typo.
thanks.
Fixed in the following commit.
https://github.com/aws/aws-sdk-rails/pull/116/commits/21c62ec8e9cfbc08a63e0c73c7333f28d4a40835